CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 
 ## [Unreleased]
+- Add a file picker dialog to choose where to export a CSV to
+
+## 4.31.0 [released 2022-??-??]
```

Comment:
i would avoid the `[released ???]` part until there's actually a release date. it's just cleaner: no release date, no release. just a version.

Reply:
it's more of a placeholder - will not merge until after release anyway